kotlin/walletsdkrs/build.gradle

```diff
@@ -104,15 +103,14 @@ publishing {
                 password = System.getenv("GITHUB_TOKEN")
             }
         }
-    }
-    publications {
-        release(MavenPublication) {
-            groupId = 'com.spruceid.wallet.sdk.rs'
-            artifactId = "walletsdkrs"
-            version = System.getenv("VERSION")
-
-            afterEvaluate {
-                from components.release
+        repositories {
+            maven {
+                name = "OSSRH"
+                url = "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
```

Comment:
can we publish to "release" directly without having to promote manually from staging (if I'm understanding things correctly)?

Reply:
doesn't seem possible